osu.Game/Rulesets/UI/DrawableRuleset.cs

```diff
@@ -496,7 +496,7 @@ public HitWindows FirstAvailableHitWindows
 
                     foreach (var n in h.NestedHitObjects)
                     {
-                        if (h.HitWindows.WindowFor(HitResult.Miss) > 0)
+                        if (n.HitWindows.WindowFor(HitResult.Miss) > 0)
```

Comment:
Can we get rid of the single letter variable names while we're at it? They were like 95% of the cause of this bug.